### PR TITLE
Teampage

### PR DESCRIPTION
**Pull request by FrankenDeba**

### CODE REVIEW

File: src/Components/BottomText/BottomText.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./BottomText.module.css";
```
Comment (FrankenDeba) [inline src/Components/BottomText/BottomText.js:2]: import structure 
1. React
2. library
3. component
4. asset
5. stylesheet

File: src/Components/BottomText/BottomText.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Divider from "../Divider/Divider";
+const BottomText = props =>{
+return (<div className = {styles.container}>
+<Divider/>
```
Comment (raunak1993) [inline src/Components/BottomText/BottomText.js:6]: Maintain proper indentation

File: src/Components/BottomText/BottomText.module.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+margin-bottom: 20px;
+}
+.container{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.first_para{
+    padding: 10px;
+    margin-bottom:26px;
```
Comment (raunak1993) [inline src/Components/BottomText/BottomText.module.css:13]: this much margin is not feasible for layouting, make the layout proper using flex

File: src/Components/BottomText/BottomText.module.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    text-align: center;
+    color:#039ee2;
+    font-size: 16px;
+}
+.second_para{
+    margin-bottom: 37px;
```
Comment (raunak1993) [inline src/Components/BottomText/BottomText.module.css:19]: same here margin too much not feasible

File: src/Components/CallUs/CallUs.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import styles from "./CallUs.module.css"
+const CallUs = () =>{
+    return (<>
+    <a href = "tell:+91123"><div className = {styles.CallUs}><i class = "fa fa-phone fa-3x"></i>Call Now</div></a>
```
Comment (raunak1993) [inline src/Components/CallUs/CallUs.js:5]: Maintain proper jsx xml format

File: src/Components/Carousel/Carousel.js [hunk 6]
```diff
@@ -25,6 +25,7 @@ class Carousel extends React.Component{
     }
     changeImage(numbers){
         
+        
         console.log(numbers[0],numbers[1],numbers[2],numbers[3],numbers[4]);
```
Comment (raunak1993) [inline src/Components/Carousel/Carousel.js:29]: remove all console.log from code

File: src/Components/Carousel/Carousel.js [hunk 7]
```diff
@@ -25,6 +25,7 @@ class Carousel extends React.Component{
...
         // if(this.props === '2'){
```
Comment (raunak1993) [inline src/Components/Carousel/Carousel.js:30]: remove commented code as well

File: src/Components/OurTeam/OurTeam.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "./OurTeam.module.css";
+import TeamDetails from "../../Components/TeamDetails/TeamDetails";
```
Comment (raunak1993) [inline src/Components/OurTeam/OurTeam.js:3]: incorrect import structure, correct it

File: src/Components/Quote/Quote.js [hunk 9]
```diff
@@ -6,13 +6,12 @@ const Quote = (props)=>{
         
             <span className = {styles.start}><i class = "fa fa-quote-left"></i></span>
             <p className = {styles.main}>
-            Leave it on Blue Waves to make any event truly memorable.
+                {props.line}
             </p>
             {/* <p className= "quote-end">
```
Comment (raunak1993) [inline src/Components/Quote/Quote.js:11]: remove commented code

File: src/Components/Quote/Quote.js [hunk 10]
```diff
@@ -6,13 +6,12 @@ const Quote = (props)=>{
         
             <span className = {styles.start}><i class = "fa fa-quote-left"></i></span>
```
Comment (raunak1993) [inline src/Components/Quote/Quote.js:7]: JSX xml format

File: src/Components/TeamDetails/TeamDetails.js [hunk 11]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "./TeamDetails.module.css";
+import ONE from "../../assets/Team/one-team.png";
+import TWO from "../../assets/Team/two-team.png";
+import THREE from "../../assets/Team/three-team.png";
+const TeamDetails = props =>{
+    return (
+        <div className = {styles.main}>
+            <div className = {styles.one}>
```
Comment (raunak1993) [inline src/Components/TeamDetails/TeamDetails.js:9]: if all the child div has same css properties then give them same class, no need of 'one', 'two', 'three'
same for oneImg, twoImg and ThreeImg give same class name

File: src/Components/TeamDetails/TeamDetails.js [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+                She is the energy centre that artistically 
+                manages the creative workload of the company, 
+                ensuring that excellent results are 
+                achieved with detailed <span className = {styles.para}>precision</span>.
+                </div>
+                
+                </div>
+             
+                
+                <div className = {styles.two}>
```
Comment (raunak1993) [inline src/Components/TeamDetails/TeamDetails.js:23]: same here

File: src/Components/TeamDetails/TeamDetails.js [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+                <img src = {TWO}/>
+                </div>
+                <div className = {styles.text}>
+                He is the power source of this successful organization,
+                with unparalleled experience of more than 15 years 
+                and extremely high standards of work ethics and <span className = {styles.para}>professionalism</span>.
+                </div>
+                </div>
+
+                <div className = {styles.three}>
```
Comment (raunak1993) [inline src/Components/TeamDetails/TeamDetails.js:34]: same here

File: src/pages/Team/index.js [hunk 14]
```diff
@@ -1,5 +1,29 @@
...
+import OurTeam from "../../Components/OurTeam/OurTeam";
+import KnowUsBetter from "../../Components/KnowUsBetter/KnowUsBetter";
+import Record from "../../Components/Record/Record";
+import BottomText from "../../Components/BottomText/BottomText";
+import Footer from "../../Components/Footer/Footer";
+import CallUs from "../../Components/CallUs/CallUs";
+
 const Team = (props) =>{
-    return (<div>this is our team</div>)
+    return (<><Header src = {LOGO}/>
```
Comment (raunak1993) [inline src/pages/Team/index.js:16]: maintain JSM xml format

**Pull request closed without merging**